### ISSUE TITLE
Eraser is the first device reported when brought near with Dell Active Pen and XPS 9575.

### ISSUE THREAD

**Issue by hperrin** [linuxwacom/xf86-input-wacom#39]
I have a pen that is reported as two devices, a stylus and an eraser. It is designed to switch to the eraser when the bottom barrel button is held. 

# Steps to reproduce

1. Bring the pen close to the screen without holding any button and move it around/touch the screen.
2. Press the eraser button and move the pen around/touch the screen.
3. Release the eraser button and move the pen around/touch the screen.

# Expected behavior

1. The device with type STYLUS reports movement/touches.
2. The device with type ERASER reports movement/touches.
3. The device with type STYLUS reports movement/touches.

# Actual behavior

1. The device with type ERASER reports movement/touches.
2. The device with type ERASER reports movement/touches.
3. The device with type STYLUS reports movement/touches.

I've tested the pen on a Surface Book with Windows 10, and it works correctly there, so I don't believe it's a hardware issue.

# Hardware

* Computer: Dell XPS 15 2 in 1 (9575)
* Pen: Dell Premium Active Pen PN579X

Relavent command output:

```
~ $ xsetwacom list
Wacom HID 486B Pen stylus               id: 12  type: STYLUS    
Wacom HID 486B Finger touch             id: 13  type: TOUCH     
Wacom HID 486B Pen eraser               id: 20  type: ERASER    
~ $ cat .config/tabletdblocalrc 
[486B]
layout=unknown
model=unknown
name=Wacom HID 486B
padbuttons=0
statusleds=0
touchring=false
touchstripl=false
touchstripr=false
wheel=false
~ $ cat .config/tabletprofilesrc 
[056A:486B]
CurrentProfileEntry=-1
ProfileRotationList=

[056A:486B][Default][eraser]
Area=0 0 0 0
Button1=1
Button2=3
Button3=2
Mode=absolute
PressureCurve=0 0 100 100
RawSample=4
Rotate=auto
ScreenSpace=desktop
Suppress=2
Threshold=27

[056A:486B][Default][stylus]
Area=0 0 0 0
Button1=1
Button2=3
Button3=2
Mode=absolute
PressureCurve=0 0 100 100
RawSample=4
Rotate=auto
ScreenSpace=desktop
Suppress=2
TabletPcButton=off
Threshold=30

[056A:486B][Default][touch]
Area=0 0 0 0
Gesture=on
InvertScroll=off
Mode=absolute
Rotate=auto
ScreenSpace=desktop
ScrollDistance=20
TapTime=250
Touch=on
ZoomDistance=50

[:][Default][eraser]
Button1=1
Button2=2
Button3=3
Mode=absolute
PressureCurve=0 0 100 100
RawSample=4
Rotate=auto
Suppress=2
Threshold=27

[:][Default][pad]
AbsWheelDown=5
AbsWheelUp=4

[:][Default][stylus]
Button1=1
Button2=2
Button3=3
Mode=absolute
PressureCurve=0 0 100 100
RawSample=4
Rotate=auto
Suppress=2
Threshold=27
```

**Comment by jigpu** [comment]
First, would you mind running our [sysinfo.sh script](https://github.com/linuxwacom/wacom-hid-descriptors/blob/master/scripts/sysinfo.sh) and attaching the generated tarball to a new [wacom-hid-descriptors issue](https://github.com/linuxwacom/wacom-hid-descriptors/issues/)? This will give us more information about your particular device and let us provide quicker help to others with the same device in the future.

Next, please run this [capture.sh script](https://gist.githubusercontent.com/jigpu/b7b47e4bede584cec8562f666fd84af4/raw/1050d9aab9514ca0658c3cb415ccb11af5e4d6d6/capture.sh) and tell it you want to record the stylus device. Perform the three steps listed in your bug report and then press CTRL+C. Then, run the script a second time, this time telling it you want to record the eraser device. Perform the three steps again and press CTRL+C afterwards. Finally, attach the tarballs generated by both the stylus and eraser runs to this bug for analysis.

**Comment by jigpu** [comment]
Believed to be resolved with pull request #92

**Comment by hperrin** [comment]
Sorry I missed the request for more info, but I'm excited to see it fixed. I'll reopen if I see the issue after the next release. Thank you! :D
